### ISSUE TITLE
Scrollbars dont interact

### ISSUE THREAD

**Issue by bascurtiz** [ohhsoash/ModioX#3]
Is it just me or are the scrollbars not functioning?
When I click them, or scroll they don't interact right?

I'm asking, since I want to use the same Dark UI Scrollbars in my project.
Have u managed to do so?

**Comment by ohhsoash** [comment]
Hi there!

Ahh yeah I see, scrollbars aren't noted in their documentation, so I needed to pull most of the code around from forums about as you mentioned in their issues, linking the scrollbar to controls and to imitate controls' scrollbars to their state.

For the current version of the app there's probably some bugs with the code. But in the next update they're all working as normal and (hopefully) for everyone else too. So as that's said, check our source code and look for something like, 

```
Update(ControlName)ScrollBars();
```

Which sets all the scrollbars properties depending on the control being used. For me it works everytime, but I did also notice that if you maximise your window then you'll most likely need to refresh the scrollbars. Listview's don't need them too.

Also, the DarkUI owner is an awesome dude and could help greatly with what we're trying to achieve! I do think I've opened something similar to this before, or least commented about it. Not sure.
